lib/tasks/turbo_scroll_tasks.rake

```diff
@@ -1,4 +1,7 @@
-# desc "Explaining what the task does"
-# task :turbo-scroll do
-#   # Task goes here
-# end
+namespace :turbo_scroll do
+  task :generate_turbo_controller do
+    source = File.join(Gem.loaded_specs["turbo-scroll"].full_gem_path, "lib", "javascript", "controllers", "turbo_controller.js")
+    target = File.join(Rails.root, "app", "javascript", "controllers", "turbo_controller.js")
+    FileUtils.cp_r source, target
```

Comment:
If we do this, we should add a check to make sure we don't overwrite an existing `turbo_controller.js`.

Also, `cp` should be enough, we're copying files, not directories, there's no need for a recursive copy.